src/models/models.ts

```diff
@@ -1,34 +1,34 @@
-export interface Contract {
-    id: string;
-    email: string;
-    endDate: string;
-    paymentFreq: PaymentFreq;
-    paymentMethod: string;
-    services: string[];
-    startDate: Date;
-    status: string;
+export interface ContractInterface {
```

Comment:
(just a neat pick)
Generally types are superior to interfaces much more flexible and less problematic, but this works ofcourse.
https://www.youtube.com/watch?v=Idf0zh9f3qQ

Reply:
Ok. But when we need some things from video we can change this to type